lib/plugins/popularity/action.php

```diff
@@ -30,7 +33,7 @@ public function register(Doku_Event_Handler $controller)
      * @param Doku_Event $event
      * @param $param
      */
-    public function autosubmit(Doku_Event &$event, $param)
+    public function autosubmit(Event &$event, $param)
```

Comment:
passing by reference of an object is not recognized by rector?

Reply:
Seems there is no rule for that, yet. I opened an issue: https://github.com/rectorphp/rector/issues/8169